### ISSUE TITLE
GRID Autosport: crash on start

### ISSUE THREAD

**Issue by qcloutier** [flathub/com.valvesoftware.Steam#420]
**Game information**
GRID Autosport

**Distribution name and version where applicable**
Gentoo Linux

**Flatpak info**
Flatpak 1.4.2
nvidia-435-17

**Problem description**
The game immediately crashes after launching. Unfortunately, there are no error messages displayed in the console whatsoever, other than a generic "Game crashed, contact Feral support". If anyone has suggestions on what I could do to try and pinpoint the issue, I'm all ears.

**Does this issue reproduce with native Steam**
No, the game runs perfectly on native Steam.

**Comment by FastOSlinux** [comment]
> **Game information**
> GRID Autosport
> 
> **Distribution name and version where applicable**
> Gentoo Linux
> 
> **Flatpak info**
> Flatpak 1.4.2
> nvidia-435-17
> 
> **Problem description**
> The game immediately crashes after launching. Unfortunately, there are no error messages displayed in the console whatsoever, other than a generic "Game crashed, contact Feral support". If anyone has suggestions on what I could do to try and pinpoint the issue, I'm all ears.
> 
> **Does this issue reproduce with native Steam**
> No, the game runs perfectly on native Steam.

Same Error Here, it happens with Shador of mordor too. need to wait update fix
https://github.com/flathub/com.valvesoftware.Steam/issues/417

**Comment by eskse** [comment]
Both of these games appear to have been ported by [Feral](http://www.feralinteractive.com/en/linux-games/) and so may be trying to use [Gamemode](https://github.com/FeralInteractive/gamemode) by default which isn't supported in the stable steam Flatpak.

It's a long shot but try running the beta version and seeing if it's the same.

**Comment by nanonyme** [comment]
That other bug was a game bug where developer built game against their own SDL2 instead of one in Steam Runtime. It had nothing to do with gamemode.

**Comment by qcloutier** [comment]
Gamemode depends on systemd, and I'm running OpenRC, so it shouldn't be a factor. I tried running the beta version anyways just for the heck of it, but it still didn't work.

**Comment by qcloutier** [comment]
So yesterday I ran a full system update, and afterwards I noticed that all the apps that I had installed via Flatpak were taking considerably longer to launch (up to a minute in some cases). I decided to try uninstalling Flatpak and all the dependencies that it had brought in, delete all directories it had created (other than ~/.var/app), and then reinstall it and all the apps. While this didn't fix the slow launch times, it somehow has made this game able to run! 

I had tried running the game after I did the updates, and it still didn't work, so it was definitely the reinstall that fixed it. In the future, I'm probably just going to blame my system itself for any issues I have with Flatpak, since it seems to be kinda flaky...
